src/setup.py

```diff
@@ -143,5 +143,12 @@ def vitables_data_files():
       ext_modules=ext_modules,
       options={"build_ext": build_ext_options, "build_exe": build_exe_options},
       executables=[Executable("main.py")],
-      requires=['numpy', 'numexpr', 'tables', 'bcolz'])
+      install_requires=['numpy', 'numexpr', 'tables'],
+      extra_requires=dict(

```

Comment:
I actually use another setup.py hacked to work on a linux machine so won't be able to test it for windows users (@AlexisEidelman ?). I do not plan to submit a complete overhaul of setup.py (with a travis CI config file etc) this early. My pint with this commit was to propagate upstream as soon as possible what could be propagated without to much hassle.
But i did checked the syntax though ;-)